### ISSUE TITLE
ncu -u causes `Warning: a promise was created in a handler but was not returned from it`

### ISSUE THREAD

**Issue by mikestopcontinues** [tjunnone/npm-check-updates#195]
`ncu -u` consistently causes this error running node 5 and most recent ncu

```
Warning: a promise was created in a handler but was not returned from it
    at printUpgrades (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:284:13)
    at /Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:184:13
From previous event:
    at analyzeProjectDependencies (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:165:6)
    at wrapper (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3533:19)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
From previous event:
    at programRunLocal (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:379:20)
    at programRun (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:328:50)
    at doNTCallback0 (node.js:407:9)
    at process._tickCallback (node.js:336:13)
From previous event:
    at Object.module.exports._.merge.run (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:400:12)
    at Object.<anonymous> (/Users/msc/.npm-packages/lib/node_modules/npm-check-updates/bin/npm-check-updates:38:5)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
    at node.js:951:3
```

**Comment by nightwolfz** [comment]
Just created my first ever pull request

https://github.com/tjunnone/npm-check-updates/pull/196

**Comment by metaraine** [comment]
Thanks! Published in v2.5.7
